### PR TITLE
WD-6078 Revamp amd-xilinx download page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update /download/amd-xilinx page

Need to be published on 19th Septemeber at 11am PST, which is about 7pm 19th of September GMT 0

## QA

- Go to https://ubuntu-com-13134.demos.haus/download/amd compare with [copy docs](https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4/edit)
    - Be sure to test on mobile, tablet and desktop screen sizes
 
- Go to https://ubuntu-com-13134.demos.haus/download/amd
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6078

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/amd/tab-3.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p>Works on:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">AMD ZCU102</li>
+        <li class="p-list__item is-ticked">AMD ZCU104</li>
+        <li class="p-list__item is-ticked">AMD ZCU106</li>
+      </ul>
+      <p><a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/zcu-ubuntu-22.04/iot-limerick-zcu-classic-desktop-2204-x05-2-20221123-58.img.xz">Download 22.04</a></p>
```
Comment (mtruj013) [inline templates/download/amd/tab-3.html:33]:
```suggestion
      <p><a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/zcu-ubuntu-22.04/iot-limerick-zcu-classic-desktop-2204-x05-2-20221123-58.img.xz">Download 22.04 LTS</a></p>
```

File: templates/download/amd/tab-3.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    </div>
+  </div>
+
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
+    <div class="col-3">
+      <h3 class="p-heading--5 u-no-padding--top"><strong>Are you working on a commercial project?</strong></h3>
+    </div>
+    <div class="col-6">
+      <p>Canonical partners with silicon vendors, board manufacturers and leading enterprises to shorten time-to-market. If you are deploying Ubuntu on AMD platforms at-scale, please reach out to <a class="js-invoke-modal" href="/download/amd-xilinx#get-in-touch">Canonical</a> to get access to ongoing bug fixes, critical security patching, long-term support; or to learn more about our solutions for custom board enablement, and application development services.</p>
```
Comment (mtruj013) [inline templates/download/amd/tab-3.html:46]:
```suggestion
      <p>Canonical partners with silicon vendors, board manufacturers and leading enterprises to shorten time-to-market. If you are deploying Ubuntu on AMD platforms at-scale, please <a class="js-invoke-modal" href="/download/amd-xilinx#get-in-touch">reach out to Canonical</a> to get access to ongoing bug fixes, critical security patching, long-term support; or to learn more about our solutions for custom board enablement, and application development services.</p>
```

File: templates/download/amd/tab-4.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<div tabindex="0" role="tabpanel" id="versal-adaptive-evaluation-kits-tab" aria-labelledby="versal-adaptive-evaluation-kits" aria-hidden="true">
```
Comment (mtruj013) [inline templates/download/amd/tab-4.html:1]: Indentation

File: templates/download/amd/tab-4.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	<div class="row p-strip is-shallow u-no-padding--top">
+		<div class="col-3 col-medium-2">
+			<h2 class="u-hide--small">Versal&trade; Adaptive SoC Evaluation Kit</h2>
```
Comment (mtruj013) [inline templates/download/amd/tab-4.html:4]:
```suggestion
			<h2 class="u-hide--small">Versal&trade;  AI Core Evaluation Kits</h2>
```

**Review by mtruj013 — APPROVED** [review]
Thanks @carkod ! Some comments in the code, mostly small copy discrepancies and indentation stuff

File: templates/download/amd/tab-4.html (hunk 3 above)
Comment (carkod) [reply templates/download/amd/tab-4.html:1]: Nothing seems to change when I change indentation in the editor... Hopefully now it does it correctly.

**Pull request merged**